integration-tests/src/test/groovy/com/okta/sdk/tests/it/util/OktaOrgCleaner.groovy

```diff
@@ -60,27 +63,75 @@ class OktaOrgCleaner {
 
         log.info("Deleting Groups:")
         client.listGroups().stream()
-                .filter { it.getProfile().getName().matches(".*-${uuidRegex}.*")}
-                .forEach {
-                    log.info("\t ${it.getProfile().getName()}")
-                    it.delete()
-                }
+            .filter { it.getProfile().getName().matches(".*-${uuidRegex}.*") }
+            .forEach {
+                log.info("\t ${it.getProfile().getName()}")
+                it.delete()
+            }
 
         log.info("Deleting Group Rules:")
         client.listGroupRules().stream()
-                .filter { it.getName().matches("rule\\+${uuidRegex}.*")}
-                .forEach {
-                    GroupRule rule = it
-                    log.info("\t ${rule.getName()}")
-                    Util.ignoring(ResourceException) {
-                        rule.deactivate()
-                    }
-                    rule.delete()
+            .filter { it.getName().matches("rule\\+${uuidRegex}.*") }
+            .forEach {
+                GroupRule rule = it
+                log.info("\t ${rule.getName()}")
+                Util.ignoring(ResourceException) {
+                    rule.deactivate()
                 }
+                rule.delete()
+            }
 
         log.info("Deleting Policies:")
         client.listPolicies(PolicyType.OKTA_SIGN_ON.toString()).stream()
-            .filter { it.getName().matches("policy\\+${uuidRegex}.*")}
+            .filter { it.getName().matches("policy\\+${uuidRegex}.*") }
+            .forEach {
+                it.delete()
+            }
+
+        log.info("Deleting LinkedObjectDefinitions:")
+        client.listLinkedObjectDefinitions().stream()
+            .forEach {
+                it.setName(it.getPrimary().getName())
+                it.delete()
+            }
+
+        log.info("Deleting InlineHooks:")
+        client.listInlineHooks().stream()
+            .forEach {
+                it.deactivate()
+                it.delete()
+            }
+
+        log.info("Deleting EventHooks:")
+        client.listEventHooks().stream()
+            .forEach {
+                it.deactivate()
+                it.delete()
+            }
+
+        log.info("Deleting UserTypes:")
+        client.listUserTypes().stream()
+            .filter { !it.getDefault() }
+            .forEach {
+                it.delete()
+            }
+
+        log.info("Deleting AuthorizationServers:")
+        client.listAuthorizationServers().stream()
+            .forEach {
+                it.deactivate()
+                it.delete()
+            }
+
+        log.info("Deleting IdentityProviders:")
+        client.listIdentityProviders().stream()
+            .forEach {
+                it.deactivate()
+                it.delete()
+            }
+
+        log.info("Deleting SmsTemplates:")
+        client.listSmsTemplates().stream()
```

Comment:
Great addition, but you probably want to filter each of these streams.
For example, this logic _might_ delete the `default` authorization server

Reply:
I checked this and it does not delete the default Authorization server.